boards/wm-200/include/periph_conf.h

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2021 Gerson Fernando Budke
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_wm-200
+ * @{
+ *
+ * @file
+ * @brief       Peripheral MCU configuration for the WM-200 Module board.
+ *
+ * @author      Gerson Fernando Budke <nandojve@gmail.com>
+ */
+#include "mutex.h"
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdint.h>
+#include <avr/io.h>
+
+#include "periph_cpu.h"
+
+/**
+ * @name    Timer peripheral configuration
+ * @{
+ */
+static const timer_conf_t timer_config[] = {
+    {
+        .dev            = (void *) &TCC1,
```

Comment:
There are 5 different Timers types on xmega:
TC0, TC1, TC2, TC4, TC5

TC0 have 4 channels. TC1 is a TC0 with only 2 channels.
TC2 is a TC0 that divide 1 16-bit timer in 2 8-bit timers. This will not be supported at first. 
TC4 is a little bit different from TC0 but with 4 channels.
TC5 is a TC4 with only 2 channels
TC4 and TC5 is only available on xmega-E variante.

I chose to implemented driver using TC0 struct definition internally and the cast is to avoid compile error.